fireblocks_sdk/sdk.py

```diff
@@ -1709,6 +1709,62 @@ def remove_fee_payer_configuration(self, base_asset):
         url = f"/v1/fee_payer/{base_asset}"
 
         return self._delete_request(url)
+    
+    def get_web3_connections(self, pageCursor=None, pageSize=None, sort=None, 
+                            filter=None, order=None):
+        """
+        Get all signer connections of the current user
+        :return: Array of sessions
+        """
+
+        method_param = locals()
+        url = "/v1/connections"
+        optional_params = ["pageCursor", "pageSize", "sort", "filter", "order"]
+
+        query_params = {param: method_param.get(param) for param in optional_params 
+                        if method_param.get(param)}
+        
+        if query_params:
+            url = url + "?" + urllib.parse.urlencode(query_params)
+        
+        return self._get_request(url)
+    
+    def create_web3_connection(self, payload, idempotency_key: str = None):
+        """
+        Initiate a new signer connection
+        :param payload: The required parameters for the connection type
+        :param idempotency_key: Idempotency key
+        :return: The created session's ID and its metadata
+        """
+
+        url = "/v1/connections/wc"
+
+        return self._post_request(url, payload, idempotency_key)
```

Comment:
lets either create a type for payload or create the object we send and not do a transparent pass through